cli/command/cli.go

```diff
@@ -79,7 +79,10 @@ type DockerCli struct {
 	dockerEndpoint        docker.Endpoint
 }
 
-var storeConfig = store.NewConfig(
+// ContextStoreConfig contains the config used by the context store
+// A plugin's init function may modify it to define other typed endpoint metadata by calling
+// ContextStoreConfig.SetEndpoint()
+var ContextStoreConfig = store.NewConfig(
```

Comment:
Wouldn't it better to have a function here ?

Reply:
(to be used from outside)

Reply:
something like `command.SetContextStoreEndpointType()` ?

Reply:
Or `store.DefaultConfig()` ? 🤔